nyxx_extensions/lib/src/embedbuilder_extension.dart

```diff
@@ -0,0 +1,59 @@
+import "package:nyxx/nyxx.dart";
+
+/// Collection of extensions for [EmbedFieldBuilder]
+extension EmbedFieldBuilderJson on EmbedFieldBuilder {
+  /// Returns a [EmbedFieldBuilder] with data from the raw json
+  EmbedFieldBuilder importJson(Map<String, dynamic> raw) {
+    this.name = raw["name"];
+    this.content = raw["value"];
+    this.inline = raw["inline"] as bool?;
+    return this;
+  }
+}

```

Comment:
it is possible to add factory constuctors through extensions, if so that woud be much better API. If that not possible then static method would be cool

Reply:
Not sure :/